code/modules/admin/topic.dm

```diff
@@ -2868,9 +2868,12 @@
 				SSblackbox.record_feedback("tally", "admin_secrets_fun_used", 1, "Traitor All ([objective])")
 
 				for(var/mob/living/carbon/human/H in GLOB.player_list)
-					if(H.stat == 2 || !H.client || !H.mind) continue
-					if(is_special_character(H)) continue
-					//traitorize(H, objective, 0)
+					if(H.stat == 2 || !H.client || !H.mind)
+						continue
+					if(is_special_character(H))
```

Comment:
This makes it so that clings can't become traitors using the secret. Prob just want to check if they already have the traitor antag datum or not instead.

Reply:
Oh yep I was blind yesterday evening. All good to leave this as is for me then!